packages/terra-clinical-item-view/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "terra-clinical-item-view",
+  "main": "lib/ClinicalItemView.js",
+  "version": "0.1.0",
+  "description": "terra-clinical-item-view",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-clinical-item-view",
+    "TerraClinicalItemView",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "csvtojson": "^1.1.4",
+    "jsdom": "^9.11.0",
+    "svgo": "^0.7.2",
+    "terra-markdown": "^0.1.0",
+    "terra-props-table": "^0.1.0",
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "terra-mixins": "^1.0.0",
+    "terra-icon": "^0.0.0"
```

Comment:
devDependency?

If it is required to use the terra-clinical-item-view component(imported in src/lib), then it should be a `dependency`. 

If we only use terra-clinical-item-view component in examples, it should be a `devDependency`.